email_account.go

```diff
@@ -0,0 +1,180 @@
+package main
+
+import (
+	"database/sql"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	emailLib "github.com/jordan-wright/email"
+)
+
+type EmailAccount struct {
+	Id        string    `json:"id,omitempty"`
+	Email     string    `json:"email"`
+	PubKey    string    `json:"pubkey,omitempty"`
+	HasPubKey bool      `json:"has_pubkey,omitempty"`
+	Created   time.Time `json:"created,omitempty"`
+}
+
+func GetEmailAccount(id string, db *sql.DB) (*EmailAccount, error) {
```

Comment:
@ajvb How about `db *sql.DB` is the first parameter for all these funcs, for consistency?